queries.yaml

```diff
@@ -9,6 +9,7 @@ all:
   - boundaries
   - transit
   - admin_areas
+  - landmarks
```

Comment:
Add docs

Reply:
See the traffic layers for inspiration for optional layer descriptions.

Reply:
ha! i was looking around for docs, it seemed like even the changelog isnt up to date. ill grep around to see if i can find them!

Reply:
https://github.com/tilezen/vector-datasource/blob/master/docs/layers.md#layer-reference
and
https://github.com/tilezen/vector-datasource/blob/master/docs/layers.md#traffic-flow

Reply:
The CHANGELOG is out of date. Historically that's been updated as releases are tagged – and it's past time to tag v1.9. Thanks for the nudge.